src/modules/endpoint/sagas/loadCities.js

```diff
@@ -23,7 +24,10 @@ function * loadCities (
 
   console.debug('Fetching cities')
 
-  const payload = yield call(() => createCitiesEndpoint(baseUrl).request())
+  const appSettings = new AppSettings()
+  const apiUrlOverride = yield call(appSettings.loadApiUrlOverride)
+
+  const payload = yield call(() => createCitiesEndpoint(apiUrlOverride || baseUrl).request())
```

Comment:
Those two cases `apiUrlOverride` is null or not should be tested in `loadCities.spec.js`.

Reply:
I tested the newly created `determineApiUrl` helper function. I'm not sure whether we still need testing here. We could now only test if we use the helper function or as far as I can see. Don't know if this makes sense.